src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.Arrays;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
```

Comment:
Consider using a more descriptive naming for the `Truck` instances instead of `truck`, `truck1`, `truck2`. Descriptive names will make the code more readable and maintainable.